common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -394,74 +394,13 @@ def test_from_xml_when_handout_is_course_asset(self, course_id_string, expected_
             'transcripts': {'uk': 'ukrainian_translation.srt', 'de': 'german_translation.srt'},
         })
 
-    def test_from_xml_missing_attributes(self):
-        """
-        Ensure that attributes have the right values if they aren't
-        explicitly set in XML.
-        """
-        module_system = DummySystem(load_error_modules=True)
-        xml_data = '''
-            <video display_name="Test Video"
-                   youtube="1.0:p2Q6BrNhdh8,1.25:1EeWXzPdhSA"
-                   show_captions="true">
-              <source src="http://www.example.com/source.mp4"/>
-            </video>
-        '''
-        output = VideoDescriptor.from_xml(xml_data, module_system, Mock())
-        self.assert_attributes_equal(output, {
-            'youtube_id_0_75': '',
-            'youtube_id_1_0': 'p2Q6BrNhdh8',
-            'youtube_id_1_25': '1EeWXzPdhSA',
-            'youtube_id_1_5': '',
-            'show_captions': True,
-            'start_time': datetime.timedelta(seconds=0.0),
-            'end_time': datetime.timedelta(seconds=0.0),
-            'track': '',
-            'handout': None,
-            'download_track': False,
-            'download_video': True,
-            'html5_sources': ['http://www.example.com/source.mp4'],
-            'data': ''
-        })
-
-    def test_from_xml_missing_download_track(self):
-        """
-        Ensure that attributes have the right values if they aren't
-        explicitly set in XML.
-        """
-        module_system = DummySystem(load_error_modules=True)
-        xml_data = '''
-            <video display_name="Test Video"
-                   youtube="1.0:p2Q6BrNhdh8,1.25:1EeWXzPdhSA"
-                   show_captions="true">
-              <source src="http://www.example.com/source.mp4"/>
-              <track src="http://www.example.com/track"/>
-            </video>
-        '''
-        output = VideoDescriptor.from_xml(xml_data, module_system, Mock())
-        self.assert_attributes_equal(output, {
-            'youtube_id_0_75': '',
-            'youtube_id_1_0': 'p2Q6BrNhdh8',
-            'youtube_id_1_25': '1EeWXzPdhSA',
-            'youtube_id_1_5': '',
-            'show_captions': True,
-            'start_time': datetime.timedelta(seconds=0.0),
-            'end_time': datetime.timedelta(seconds=0.0),
-            'track': 'http://www.example.com/track',
-            'download_track': True,
-            'download_video': True,
-            'html5_sources': ['http://www.example.com/source.mp4'],
-            'data': '',
-            'transcripts': {},
-        })
-
```

Comment:
Why were the above tests removed?

Reply:
I initially thought they were related to the `source` field but looking at it again, with a small change, we can retain the rest of the test. So I have added it back. Thanks for catching this!